content/sse/index.textile

```diff
@@ -140,15 +140,15 @@ Start a streaming HTTP request, for ease of consuming with any HTTP library that
-- rewind := (optional integer) if specified, will send a backlog of the number of messages specified once the connection opens. For example, @rewind=1@ will give you the most recent message sent on the channel. This is best-effort — only messages from within the last two minutes will be available, and only if the channel has been continuously active since the message was sent; it is not a replacement for the "history API":/rest/history. It only has an effect for new connections; when resuming a previous connection using @lastEvent@, it is ignored in favour of sending you the messages you missed since you were last connected.
-- enveloped := (optional, default true) if true, the @data@ from each event envelope for a @message@ event will be a "Message":/realtime/types#message object. If false, it will be the payload from the message directly. See "Envelope format":#envelope-format below.
-- heartbeats := (optional, default false) if @true@ will use an explicit heartbeat event rather than a newline as a keepalive packet.
+- channels := **mandatory**. One or more channel names, separated by commas (or the @separator@ if specified). Non-url-safe characters should be URL-encoded (for example, @?channels=foo%3Fbar@ will subscribe to the channel @foo?bar@). Alias: @channel@.
+- v := **mandatory**. The version of the api you are requesting. The current (and only) version of the API is 1.1, so only @v=1.1@ is accepted.
+- separator := **optional**. A separator, to enable easy subscriptions to channels with commas in their name. For example, @?separator=|&channel=fo,o|ba,r@ will subcribe to the two channels @fo,o@ and @ba,r@.
+- key := **optional**. An Ably API key to use, if using basic auth.
+- token := **optional** An Ably auth token to use, if using token auth.
+- lastEvent := **optional**. An @id@ to resume from. Only required when starting a new SSE connection which resumes from a previous connection.
+- rewind := **optional**. An integer which, if specified, will send a backlog of the number of messages specified once the connection opens. For example, @rewind=1@ will give you the most recent message sent on the channel. This is best-effort — only messages from within the last two minutes will be available, and only if the channel has been continuously active since the message was sent; it is not a replacement for the "history API":/rest/history. It only has an effect for new connections; when resuming a previous connection using @lastEvent@, it is ignored in favour of sending you the messages you missed since you were last connected.
+- enveloped := **optional**. Default is **true**. If true, the @data@ from each event envelope for a @message@ event will be a "Message":/realtime/types#message object. If false, it will be the payload from the message directly. See "Envelope format":#envelope-format below.
+- heartbeats := ≈ if @true@ will use an explicit heartbeat event rather than a newline as a keepalive packet.
```

Comment:
Why is this one not flagged as optional?

Reply:
Fixed with https://github.com/ably/docs/pull/666/commits/2e53135582c4897a94af7a2cc966669c7dbf20c8.